packages/text-field/Input.tsx

```diff
@@ -25,22 +25,20 @@ import * as classnames from 'classnames';
 import {MDCTextFieldFoundation} from '@material/textfield/dist/mdc.textfield';
 
 export interface InputProps<T> {
-  className: string;
-  inputType: 'input' | 'textarea';
-  disabled: boolean;
```

Comment:
Should this have been removed ?  It looks like it's still referenced.

Reply:
Nope!

Reply:
OH...yes I meant to remove disabled and id, because its duplicated in `type Props<T> = InputProps<T> & (T extends HTMLInputElement ? InputElementProps : TextareaElementProps);`